plugin/evm/gossiper.go

```diff
@@ -185,15 +186,50 @@ func (n *pushGossiper) queueRegossipTxs() types.Transactions {
 		)
 		return nil
 	}
-	localQueued := n.queueExecutableTxs(state, tip.BaseFee(), localTxs, n.config.TxRegossipMaxSize)
+	rgFrequency := n.config.RegossipFrequency
+	rgMaxTxs := n.config.RegossipMaxTxs
+	localQueued := n.queueExecutableTxs(state, tip.BaseFee(), localTxs, rgFrequency, rgMaxTxs, 1)
 	localCount := len(localQueued)
-	if localCount >= n.config.TxRegossipMaxSize {
+	if localCount >= rgMaxTxs {
 		return localQueued
 	}
-	remoteQueued := n.queueExecutableTxs(state, tip.BaseFee(), remoteTxs, n.config.TxRegossipMaxSize-localCount)
+	remoteQueued := n.queueExecutableTxs(state, tip.BaseFee(), remoteTxs, rgFrequency, rgMaxTxs-localCount, 1)
```

Comment:
nit: would read nicer if we had `maxTxPerAccount := 1` and pass that or a comment

Reply:
I'm just going to make this another config